### PR TITLE
Support pasting URLs over markdown text

### PR DESCRIPTION
**Pull request by silverwind**
Support pasting URLs over selection text in the textarea editor. Does not work in EasyMDE and I don't intend to support it. Image paste works as usual in both Textarea and EasyMDE.

The new `replaceTextareaSelection` function changes textarea content via [`insertText`](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand#using_inserttext) command, which preserves history, e.g. `CTRL-Z` works and is also demostrated below. We should later refactor the image paste code to use the same function because it currently destroys history.

Overriding the formatting via `Shift` key is supported as well, e.g. `Ctrl+Shift+V` will insert the URL as-is, like on GitHub.

urlpaste

### CODE REVIEW

**Comment by silverwind** [comment]
I think this can be barely called a feature in our generally understood sense of backend-dominant features 😆.

**Review by techknowlogick — APPROVED** [review]

File: web_src/js/features/comp/Paste.js [hunk 1]
```diff
@@ -131,18 +118,37 @@ const uploadClipboardImage = async (editor, dropzone, e) => {
...
-    return uploadClipboardImage(new CodeMirrorEditor(easyMDE.codemirror), dropzone, e);
+  easyMDE.codemirror.on('paste', (_, e) => {
+    const {images} = getPastedContent(e);
+    if (images.length) {
+      handleClipboardImages(new CodeMirrorEditor(easyMDE.codemirror), dropzone, images, e);
+    }
   });
 }
 
 export function initTextareaImagePaste(textarea, dropzone) {
```
Comment (KN4CK3R) [inline web_src/js/features/comp/Paste.js]: The name doesn't match the functionality anymore.

Comment (silverwind) [reply web_src/js/features/comp/Paste.js]: Ah, yes these need to rename to remove `Image`

Comment (silverwind) [reply web_src/js/features/comp/Paste.js]: Done.

**Comment by silverwind** [comment]
Ready again.

**Review by KN4CK3R — APPROVED** [review]

**Pull request merged**